linux_os/guide/system/software/sudo/sudo_add_ignore_dot/rule.yml

```diff
@@ -0,0 +1,39 @@
+documentation_complete: true
+
+prodtype: rhel7,rhel8
+
+title: 'Ensure sudo Ignores Commands In Current Dir - sudo ignore_dot'
+
+description: |-
+    The sudo <tt>ignore_dot</tt> tag, when specified, will ignore the current directory
+    in the PATH environment variable.
+    On {{{ full_name }}}, <tt>env_reset</tt> is enabled by default
```

Comment:
I am not sure that we can guarantee this across everything. Maybe drop the line?